### DISCUSSION TITLE
HM-ES-TX-WM Auswerteroutinen

### DISCUSSION THREAD

**Discussion by alange6635** [OpenCCU/OpenCCU#1772] [🇩🇪 Fragen&Antworten] [answered]
Hallo Jens,
das war einmal eine Idee von Dir.(11.2016) 
(Bin auch am überlegen ob ich vielleicht sogar ein komplettes CCU Addon nur für den HM-ES-TX-WM entwickle mit dem man dann alle seine Hauszähler entsprechend managen kann und dann solche Sachen wie werte resetten, etc. komplett über die Webseite des Addons managen kann. Aber ganz so schlüssig bin ich da nicht ob ich mir die ganze Arbeit wirklich machen sollte.) Ist doch eine tolle Sache, gibt es da Fortschritte? Kann man Dich motivieren dazu? Ich hoffe das kommt, natürlich mit Kostenbeteiligung.

**Comment by jens-maus** [comment, marked as answer]
Dafür habe ich ehrlich gesagt selbst keine Zeit da in irgendwelche Auswerteroutinen für den HM-ES-TX-WM Arbeit reinzustecken. Sowas sollte - und muss - aus der Community heraus passieren. Wenn du also ein eigenes Addon oder so dafür schreiben willst, da mach das. Von mir wird diesbzgl. nichts kommen.

Und natürlich gibt es übrigens folgende von mir vor vielen Jahren entwickelte ReGa Skripte die auch heutzutage noch prächtig bei mir funktionieren:

https://homematic-forum.de/forum/viewtopic.php?t=31353

**Comment by alange6635** [comment]
Danke für Deine Antwort. Schade! Um so ein Addon zu schreiben braucht man ein Specialist und der bist Du! Bin gespannt wieder etwas Neues von Dir zu hören. Alles Gute.
